domains/kenbanks.json

```
@@ -0,0 +1,9 @@
+{
+	"owner": {
+		"username": "kenbankspeng",
+		"email": "kenbanks.peng@gmail.com"
+	},
+	"record": {
+		"CNAME": "kenbankspeng.github.io/ken"
```

CNAME can't contain `/`
```suggestion
		"CNAME": "kenbankspeng.github.io"
```